test/user_app_logging_test.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // Logging tests define the ability for Cloud Platform to perform aggregated logging
 // on the platform. The tests are designed to be run in a Kubernetes cluster, with a logging agent installed.
-var _ = Describe("logging", Ordered, func() {
+var _ = FDescribe("logging", Ordered, func() {
```

Comment:
I think you need to get rid of the `F` here?

Reply:
oops!